hbase-balancer/src/main/java/org/apache/hadoop/hbase/master/balancer/SlopFixingCandidateGenerator.java

```diff
@@ -63,6 +64,13 @@ BalanceAction generateCandidate(BalancerClusterState cluster, boolean isWeighing
     List<MoveRegionAction> moves = new ArrayList<>();
     Set<ServerAndLoad> fixedServers = new HashSet<>();
     for (int sourceServer : sloppyServerIndices) {
+      if (
+        isTableIsolationEnabled
+          && getBalancerConditionals().isServerHostingIsolatedTables(cluster, sourceServer)
+      ) {
+        // Don't fix sloppiness of servers hosting isolated tables
```

Comment:
Do we have a hook for excluding such servers from the sloppiness metric? I.e., if the only slop is from this case, will the slop metrics continue to be non-zero?

Reply:
We don't have anything similar for metrics. I'll think about whether that would be difficult to implement

Reply:
Actually I don't think there's a metric reported here